prometheus-exporter/script/prometheus_chart_exporter.py

```diff
@@ -1,86 +1,57 @@
 import argparse
 import io
 import json
-import re
 import requests
 import sys
 import shlex
 import shutil
 import plotly.express as px
```

Comment:
Ya no es necesario importar Shlex, ya que has quitado la única vez que se utilizaba (para hacer un split)

```suggestion
import requests
import sys
import shutil
import plotly.express as px
```